packages/sdk-metrics/src/state/TemporalMetricProcessor.ts

```diff
@@ -67,15 +67,15 @@ export class TemporalMetricProcessor<T extends Maybe<Accumulation>> {
    * Builds the {@link MetricData} streams to report against a specific MetricCollector.
    * @param collector The information of the MetricCollector.
    * @param collectors The registered collectors.
-   * @param instrumentDescriptor The instrumentation descriptor that these metrics generated with.
+   * @param descriptor The instrumentation descriptor that these metrics generated with.
```

Comment:
```suggestion
   * @param descriptor The metric descriptor that these metrics generated with.
```

Reply:
Good catch :slightly_smiling_face: 
fixed in [dd7a25f](https://github.com/open-telemetry/opentelemetry-js/pull/3520/commits/dd7a25fa92bd67fe892a6b04d7dbc8609f2972ae) :slightly_smiling_face: